File: Data Structures & Algorithms/append-characters-to-string-to-make-subsequence/submission-0.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        i, j = 0, 0

        while i < len(s) and j < len(t):
            if s[i] == t[j]:
                j+=1
            i+=1
        
        if j == len(t):
            return 0
        return len(t)-(j)
        

File: Data Structures & Algorithms/search-2d-matrix/submission-0.java
class Solution {
    public boolean searchMatrix(int[][] matrix, int target) {
        for(int row = 0; row < 3; row++){
            for(int col = 0; col < 4; col++){
                if(matrix[row][col] == target){
                    return true;
                }
            }
        }
        return false;
    }
}
